index.html

```diff
@@ -21,44 +21,44 @@
   </head>
   <body>
     <!-- avatar -->
-    <h1>avatars</h1>
+    <!-- <h1>avatars</h1> -->
 
-    <div class="avatar avatar-xs">
+    <!-- <div class="avatar avatar-xs">
       <img
-        src="https://thumbor.forbes.com/thumbor/711x711/https://specials-images.forbesimg.com/imageserve/613df8e8d679a21b766a1636/bigbun-2/960x0.jpg?fit=scale"
+        src="components/images/avatar-img.jpeg"
         alt="avatar-img"
         srcset="
-          https://thumbor.forbes.com/thumbor/711x711/https://specials-images.forbesimg.com/imageserve/613df8e8d679a21b766a1636/bigbun-2/960x0.jpg?fit=scale
+          components/images/avatar-img.jpeg
         "
         class="img-round img-responsive"
       />
     </div>
     <div class="avatar avatar-sm">
       <img
-        src="https://thumbor.forbes.com/thumbor/711x711/https://specials-images.forbesimg.com/imageserve/613df8e8d679a21b766a1636/bigbun-2/960x0.jpg?fit=scale"
+        src="components/images/avatar-img.jpeg"
         alt="avatar-img"
         srcset="
-          https://thumbor.forbes.com/thumbor/711x711/https://specials-images.forbesimg.com/imageserve/613df8e8d679a21b766a1636/bigbun-2/960x0.jpg?fit=scale
+          components/images/avatar-img.jpeg
         "
         class="img-round img-responsive"
       />
     </div>
     <div class="avatar avatar-md">
       <img
-        src="https://thumbor.forbes.com/thumbor/711x711/https://specials-images.forbesimg.com/imageserve/613df8e8d679a21b766a1636/bigbun-2/960x0.jpg?fit=scale"
+        src="components/images/avatar-img.jpeg"
         alt="avatar-img"
         srcset="
-          https://thumbor.forbes.com/thumbor/711x711/https://specials-images.forbesimg.com/imageserve/613df8e8d679a21b766a1636/bigbun-2/960x0.jpg?fit=scale
+          components/images/avatar-img.jpeg
         "
         class="img-round img-responsive"
       />
     </div>
     <div class="avatar avatar-lg">
       <img
-        src="https://thumbor.forbes.com/thumbor/711x711/https://specials-images.forbesimg.com/imageserve/613df8e8d679a21b766a1636/bigbun-2/960x0.jpg?fit=scale"
+        src="components/images/avatar-img.jpeg"
         alt="avatar-img"
         srcset="
-          https://thumbor.forbes.com/thumbor/711x711/https://specials-images.forbesimg.com/imageserve/613df8e8d679a21b766a1636/bigbun-2/960x0.jpg?fit=scale
+          components/images/avatar-img.jpeg
         "
         class="img-round img-responsive"
       />
```

Comment:
you may add loading="lazy" as these badges need not load immediately with the doc site.

Reply:
okay will make the change